coderd/coderdtest/authtest.go

```diff
@@ -422,32 +445,18 @@ func (a *AuthTester) Test(ctx context.Context, assertRoute map[string]RouteCheck
 			}
 			a.t.Run(name, func(t *testing.T) {
 				a.authorizer.reset()
-				routeAssertions, ok := assertRoute[strings.TrimRight(name, "/")]
+				routeKey := strings.TrimRight(name, "/")
+				routeAssertions, ok := assertRoute[routeKey]
 				if !ok {
 					// By default, all omitted routes check for just "authorize" called
```

Comment:
I think we need to drop this comment then? This was initially done because I was lazy, but I totally see how it allowed some routes to drift :man_facepalming:

Reply:
I still think it's fine for routes in the walk not to be in `assertRoute` and us just check that auth is called.

The issue is that there were entries in the `assertRoute` that were not present in the walk.  Those entries asserted particular RBAC actions and objects, but because the routes were misspelled, the assertions were never checked.